doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -5,21 +5,91 @@
 
 ## Table of Contents
 * [1.1 Introduction.](#1.1)
-* [1.2 Terminology.](#1.2)
-* [1.3 Approach](#1.3)
-* [1.4 Principles](#1.4)
-* [1.5 scope](#1.4)
-* [1.6 Vision](#1.4)
-* [1.7 Roadmap](#1.4)
+* [1.2 Terminology](#1.2)
+* [1.3 Principles](#1.3)
+  * [1.3.1 Cloud Native.](#1.3.1)
+* [1.4 Scope](#1.4)
+* [1.5 Vision](#1.5)
+* [1.6 Approach](#1.6)
+* [1.7 Roadmap](#1.7)
+
 
 <a name="1.1"></a>
 ## 1.1 Introduction
 
-This document (at this stage, not a RA, we're just using the template as a placeholder), discusses the role of Kubernetes in managing NFV software components (e.g. VNFCs). Application containers such as those managed by Kubernetes can be considered a Foundational PaaS capability<sup>1</sup> and so the following terms can all be thought of as being the topic of this document:
-- Kubernetes PaaS (Platform-as-a-Service)
-- CaaS (Containers-as-a-Service)
-- Kubernetes-based CaaS (Containers-as-a-Service)
-- Kubernetes-as-a-Service
+> to be written
+
+
+<a name="1.2"></a>
+### 1.2 Terminology
```

Comment:
Ensuring consistency between terms is hard. IMO, it helps taking them from a single source rather than mixing them from several sources. The k8s glossary contains many of the terms (container, cluster, pod, ...) we need:
https://kubernetes.io/ja/docs/reference/glossary/?fundamental=true